lumigator/python/mzai/backend/backend/ray_submit/submission.py

```diff
@@ -16,7 +16,7 @@ class RayJobEntrypoint(ABC):
     memory: int | float | None = None
 
     @property
-    def command_with_params(self) -> str:
+    def get_command_with_params(self) -> str:
```

Comment:
changing this to a verb since we generally want methods to be verbs and variables nouns